Libplanet.Tests/Net/Protocols/TestTransport.cs

```diff
@@ -68,47 +67,69 @@ public class TestSwarm : ISwarm
 
         internal IProtocol Protocol { get; }
 
-        internal bool Running => !(swarmCancellationTokenSource is null);
+        internal bool Running => !(_swarmCancellationTokenSource is null);
 
         public void Dispose()
         {
         }
 
-        public async void Start()
+        public async Task StartAsync(
+            CancellationToken cancellationToken = default(CancellationToken))
         {
-            _logger.Debug("Starting swarm {Peer}.", AsPeer);
-            swarmCancellationTokenSource = new CancellationTokenSource();
-            await ProcessRuntime(swarmCancellationTokenSource.Token);
+            _logger.Debug("Starting transport of {Peer}.", AsPeer);
+            _swarmCancellationTokenSource = new CancellationTokenSource();
+            await Task.Delay(10, cancellationToken);
```

Comment:
Why is this delay needed?

Reply:
I added it because `ITransport` needs `RunAsync()` to be `async`, but `TestTransport`'s `RunAsync()` need not to be `async`.